File: Wayne/WayneCompiler/src/Dfir_representation/Connection.java
package Dfir_representation;

public class Connection{
	   private int terminalId;
	   private int nodeId;
	   public int getTerminalId() {
		return terminalId;
	}
	public void setTerminalId(int terminalId) {
		this.terminalId = terminalId;
	}
	public int getNodeId() {
		return nodeId;
	}
	public void setNodeId(int nodeId) {
		this.nodeId = nodeId;
	}
	public Connection(int TerminalId, int NodeId){
		   this.terminalId = TerminalId;
		   this.nodeId = NodeId;
	   }
	   public void print(){
		   
		   System.out.println("TerminalId: " + this.terminalId + ' ' + "NodeId: " + this.nodeId);
		   
	   }
}

File: Jasmine/JasmineCompiler/Src/DFIR_Compiler/DFIR_Compiler/Connection.cs
using System;

namespace DFIR_Compiler
{
    public class Connection
    {
        private int nodeId;
        private int terminalId;
        public Connection(int TerminalId, int NodeId)
        {
            terminalId = TerminalId;
            nodeId = NodeId;
        }

        public virtual int GetNodeId()
        {
            return nodeId;
        }
        public virtual int GetTerminalId()
        {
            return terminalId;
        }


        public virtual void SetNodeId(int NodeId)
        {
            nodeId = NodeId;
        }
        public virtual void SetTerminalId(int TerminalId)
        {
            terminalId = TerminalId;
        }


        public virtual void Print()
        {
            Console.WriteLine("TerminalId: " + terminalId + ' ' + "NodeId: " + nodeId);
        }

    }
}
